Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.8.3 as builder
```

Comment:
`go1.9` has been released. It may be good to add `1.9` as a version to test in `.travis.yml` and to then use `go1.9` here when building the software.